sips/minutes/_posts/2016-11-29-sip-minutes.md

```diff
@@ -0,0 +1,97 @@
+
+The spec needs to be updated based on Martin's Dotty implementation. We need to split the SIPs for the next meeting.
+
+**Conclusion** More development needed. Must answer questions on PR.
+
+### [SIP-24 - Repeated By Name Parameters](http://docs.scala-lang.org/sips/pending/repeated-byname.html)
+
+Heather says the debate is about the semantics or translation rules. All arguments are evaluated each time the parameter is referenced in the method. This is implemented in Dotty. Should this be implemented in Scalac?
+
+Sébastien thinks that Java bridge should be forbidden in this case. It's annotation that we add, so we could just make it an error.
+
+The SIP needs more examples. Martin says it's to remove an annoying restriction; there's no reason we can't we mix repeated and by name parameters. Evaluating them all together is much simpler. There's another reason regarding case classes. Inline function should maintain call by value semantics. Pass parameters by name just to be clean.
+
+The main motivation is to prepare for inline. Inline won't work very well without this. Need to flesh this out in SIP.
+
+**Conclusion** Someone needs to flesh out the SIP and how it relates to inline.
+
+### [SIP-NN:Static](https://github.com/scala/scala.github.com/pull/491/files)
+
+Iulian says too much code is generated by annotations. We could solve name clashes the way ScalaJS does by specifying the exported name. How can we wake code generation predictable without looking at annotations? How do we emit public static field without accessors? Having everything emitted as static and object where possible is going to simplify reasoning about how things are initialized.
+
+How should a user decide when to use static? It is platform-dependent.
+
+This would generate another set of accessors. It changes the bytecode. It has a special relationship with lazy vals. Maybe it shouldn't be an annotation.
+
+Seth thinks it ought to be an annotation because it's affecting an external representation of the code and not the meaning from Scala's perspective.
+
+Martin says it's important that there are these restrictions, that they come first in order not to have surprises with initialization order.
+
+A lot of people are surprised about initialization order.
+
+Sometimes we need something to be static for Java interop.
+
+Sébastien says binary compatibility is also an argument in favor of having explicit @static annotation. If in one version you have a static method, in the next version you add a method with same name and signature. The static implementation is not there anymore, then you have broken binary compatibility in a silent way.
+
+**Conclusion** There are a lot of edge cases when not using annotations. We need to work on this and think about cases around initialization to simplify it. Why do statics need to go first? Show surprising results. There are different implementation strategies. Is this more like @tailrec or does it change generated code?
```

Comment:
Can we add "This is the first review iteration of this proposal." at the beginning of the conclusion?